Snowflake permission note (#6573)
dbt-labs/docs.getdbt.com: website/docs/docs/cloud/connect-data-platform/connect-snowflake.md

sidebar_label: "Connect Snowflake"
---

:::note

> A crucial part of working with dbt atop Snowflake is ensuring that users in development environments and/or service accounts in deployment to production environments have the correct permissions to take actions on Snowflake.

Reviewer: i'm not sure if we need this intro here
```suggestion
dbt Cloud works with Snowflake by inheriting permissions directly from your Snowflake data platform configuration. This ensures you have full control over what dbt Cloud can access. You can customize roles and permissions in Snowflake to fit your company's requirements and manage access to database objects. 
```

Author: That's the original not deleted below. I'm ok with removing it, but we should be careful with this language. We don't want to inadvertently indicate that Snowflake user permissions can somehow be assigned to users in dbt Cloud itself (for their permissions over the dbt account). How about:

```suggestion
dbt Cloud connections and credentials inherit the permissions of the accounts configured. You can customize roles and associated permissions in Snowflake to fit your company's requirements and fine-tune access to database objects in your account. See [Snowflake permissions](/reference/database-permissions/snowflake-permissions) for more information about customizing roles in Snowflake.
```

> You can customize roles and associated permissions in Snowflake to fit your company's requirements and fine-tune access to database objects in your account. See [Snowflake permissions](/reference/database-permissions/snowflake-permissions) for more information about customizing roles in Snowflake.

Reviewer: ```suggestion
Refer to [Snowflake permissions](/reference/database-permissions/snowflake-permissions) for more information about customizing roles in Snowflake.
```

> The roles and associated permissions in Snowflake can be customized to fit your company's requirements and fine-tune access to database objects in your account. See [Snowflake permissions](/reference/database-permissions/snowflake-permissions) for more information about customizing roles in Snowflake.

Author: ```suggestion
You can customize roles and associated permissions in Snowflake to fit your company's requirements and fine-tune access to database objects in your account. See [Snowflake permissions](/reference/database-permissions/snowflake-permissions) for more information about customizing roles in Snowflake.
```
